resources/views/layouts/app.blade.php

```diff
@@ -169,7 +169,7 @@ class="fas fa-tools"></i> {{__('menu.admin')}}</a>
                     <span class="footer-nav-link">/ <a href="{{ route('blog.all') }}">{{ __('menu.blog') }}</a></span>
                 </p>
                 <p class="mb-0">{!! __('menu.developed') !!}</p>
-                <p class="mb-0">&copy; 2019 Tr&auml;welling</p>
+                <p class="mb-0">&copy; 2020 Tr&auml;welling</p>
```

Comment:
Maybe it would be better to change the year to {{date('Y')}} to prevent these changes.
Even if this is not the primary part of the PR.